### PR TITLE
Terms and Conditions page

### PR DESCRIPTION
**Pull request by avikalpg**
- Added the MDX library to the project and set it up (note that this still comes up as an experimental feature @tapishr )
- Created the `terms.mdx` file using ChatGPT 3.5 -> this is used inside the `terms.tsx` file

### CODE REVIEW

File: mdx-components.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+export function useMDXComponents(components: MDXComponents): MDXComponents {
+	return {
+		// Allows customizing built-in components, e.g. to add styling.
+		h1: ({ children }) => <h1 className='mt-4 mb-2 font-bold text-4xl'>{children}</h1>,
+		h2: ({ children }) => <h2 className='mt-4 mb-1 text-3xl font-semibold'>{children}</h2>,
+		h3: ({ children }) => <h3 className='mt-2 mb-1 text-2xl font-extrabold'>{children}</h3>,
+		h4: ({ children }) => <h4 className='mt-2 text-xl font-semibold'>{children}</h4>,
+		a: ({ href, target, children }) => <Link href={href ? href : "#"} target={target} className='m-0 text-primary-main'>{children}</Link>,
+		...components,
+	};
```
Comment (tapishr) [inline mdx-components.tsx:19]: @avikalpg - Can you help me understand what's going on here?

File: pages/terms.mdx [hunk 2]
```diff
@@ -0,0 +1,41 @@
+Terms and Conditions of Use
+===
+
+Welcome to Vibinex! These terms and conditions ("Terms") govern your use of the Vibinex website ("Website") and any services, features, or content provided by Vibinex ("Service"). By accessing or using the Website and Service, you agree to be bound by these Terms. If you do not agree to these Terms, please refrain from using the Website and Service.
+
+## Description of Service
+Vibinex is a privacy-first open-source tool designed to assist developers in distributing and personalizing their pull request review process on platforms such as GitHub and Bitbucket. Our Service ensures that we do not access your codebase directly. Instead, we utilize an open-source tool integrated into your CI pipeline, which processes and SHA-encrypts specific information from your code to provide insights.
```
Comment (tapishr) [inline pages/terms.mdx:7]:
```suggestion
Vibinex is a privacy-first open-source tool designed to assist developers in distributing and personalizing their pull request review process on platforms such as GitHub and Bitbucket. Our Service ensures that we do not access your codebase directly. Instead, we utilize an open-source tool, which processes and SHA-encrypts specific information from your code to provide insights.
```

Comment (tapishr) [reply pages/terms.mdx:7]: Our packaged solution might not be necessarily a part of the ci pipeline

File: mdx-components.tsx (hunk 1 above)
Comment (avikalpg) [reply mdx-components.tsx:19]: Due to tailwind CSS, all the default styling through HTML (webpack) is disabled. So we have to define the styling for every markdown-to-html element in this file.

(read the comment before these lines)

**Review by tapishr — APPROVED** [review]

**Pull request merged**